NEWS.rst

```diff
@@ -1,3 +1,23 @@
+2.1.0 (2019-01-17)
```

Comment:
I like the date to match when it gets published to PyPI to avoid confusion.

I'll be unavailable most of next week. Could someone else determine when they'd be able to publish and update the date accordingly? @PyCQA/pyflakes-dev

Reply:
agreed, I initially had 2019-01-??, I probably should have just left it like that